x-pack/test/functional/services/ml/job_selection.ts

```diff
@@ -23,5 +23,18 @@ export function MachineLearningJobSelectionProvider({ getService }: FtrProviderC
         `Job selection should display jobs or groups '${jobOrGroupIds}' (got '${actualJobOrGroupLabels}')`
       );
     },
+
+    async assertJobSelectionNotContains(jobOrGroupIds: string[]) {
```

Comment:
The `contain` assertion doesn't extract the values of an array, i.e. `expect(['a', 'b', 'c', 'd']).to.contain(['a']);` fails. As a result, the negative assertion in this method will always succeed. I think we should make the parameter a singular: `jobOrGroupId: string`.

Reply:
Sorry I missed this part. Updated here https://github.com/elastic/kibana/pull/84594/commits/c95a44a829266d4734e239361a1a3f5f5ddb0216.